### PR TITLE
Change in itransfer lz priority

### LINKED ISSUE #35: snappy-itransfer does not re-use existing landing zones
**Describe the bug**
Consecutive calls to the `snappy-itransfer` commands create multiple landing zones.

**To Reproduce**
Consecutively call `snappy-itransfer` multiple times.

**Expected behavior**
The commands should re-use existing landing zones from the current user.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by eudesbarbosa**
closes #35 

**Summary** 

Changed `itransfer` Landing Zone priority: now the automatic behaviour is to first looks 
for active Landing Zones before creating a new one.

**Tests**

1. There were no additional error nor warning messages while using the automatic `pytest`.
2. Passed manual tests in sandbox tests ([logs](https://github.com/bihealth/cubi-tk/files/6048485/35_cubi-tk_tests.log)).

### CODE REVIEW

**Comment by holtgrewe** [comment]
Is this  still a draft? If so please convert it to draft.

**Comment by eudesbarbosa** [comment]
No, it should be good to go. It wasn't a big change.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-sample_info.yaml
```
Comment (holtgrewe) [inline .gitignore:1]: The unit tests create these, this should stayl.

File: cubi_tk/snappy/itransfer_common.py [hunk 2]
```diff
@@ -369,9 +369,13 @@ def get_sodar_info(self):
 
             if create_lz_bool:
                 # Assume that provided UUID is associated with a Project and user wants a new LZ.
-                # Behavior: create new LZ.
+                # Behavior: search for available LZ; if none,create new LZ.
                 try:
-                    lz_uuid, lz_irods_path = self.create_landing_zone(project_uuid=in_destination)
+                    lz_uuid, lz_irods_path = self.get_latest_landing_zone(project_uuid=in_destination)
+                    if not lz_irods_path:
+                        logger.info(f"No active Landing Zone available for project {lz_uuid}, "
```
Comment (holtgrewe) [inline cubi_tk/snappy/itransfer_common.py:376]: Please use the common logging style of not using format strings but `"format %s"` strings with arguments afterwards.

**Review by holtgrewe — CHANGES_REQUESTED** [review]
Generally looks good, please address change.

File: cubi_tk/snappy/itransfer_common.py (hunk 2 above)
Comment (eudesbarbosa) [reply cubi_tk/snappy/itransfer_common.py]: Updated.

**Comment by holtgrewe** [comment]
LGTM, somehow CI is not run if this comes from another repo. Could you work on a branch on this repo next time? Or maybe you could look into why CI is not running? Probably a setting... Have you resolved everything? Can you mark the requested changes as resolved to make it explicit?

**Comment by holtgrewe** [comment]
You're the man!

**Review by holtgrewe — APPROVED** [review]

**Comment by holtgrewe** [comment]
Either squash changes or do a manual `rebase -i` to split this into CI and content. Then perform a rebase commit.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -36,6 +36,7 @@ jobs:
         run: coverage report
       - name: Create XML report for codacy
         run: coverage xml
+        if: github.ref == 'refs/heads/master'
```
Comment (eudesbarbosa) [inline .github/workflows/ci.yml:39]: @holtgrewe , does this look right to you?

Comment (holtgrewe) [reply .github/workflows/ci.yml:39]: Ah, this has to be used at the point where the report is posted, which is probably below that, extend line 44/45.

Comment (eudesbarbosa) [reply .github/workflows/ci.yml:39]: I don't get it. Can you maybe add a code review?

Comment (holtgrewe) [reply .github/workflows/ci.yml]: OK, added the patch (quite nice with the web IDE of Github)

**Pull request merged**